Australian Digital Observatory/content/the-ado-ecosystem/contents.lr

```diff
@@ -15,3 +15,35 @@ The ADO ecosystem aims to equip researchers with a set of modular, open source,
 This modular ecosystem approach gives flexibility, allowing researchers to pick and choose [resources](../resources) that are relevant and suitable for them, without being locked in to a specific stack (e.g., proprietary formats/tools, specific cloud infrastructure). The modular approach also recognises different entry levels in terms of researchers’ skills and needs. Furthermore, smaller tools are easier to maintain, and focusing on modularity and interoperability allows making use of and supporting existing tools and resources already in the community rather than reinventing the wheel.
 
 In designing and curating the ADO ecosystem, we take inspiration from the open source software community and [the Unix philosophy](https://medium.com/ingeniouslysimple/philosophy-of-unix-development-aa0104322491) of designing interoperable tools that each do one thing well, and from foundational tooling ecosystems such as [the R Tidyverse](https://rviews.rstudio.com/2017/06/08/what-is-the-tidyverse/#:~:text=The%20tidyverse%20is%20a%20coherent,being%20expanded%20by%20several%20contributors.), which has well demonstrated the suitability of such a structure to the academic research environment. We hope to learn from these communities, and also to contribute back to them.
+
+### Example workflows
+
+Here are some examples of research workflows that benefit from our ecosystem approach. 
+
+**Twitter conversation analysis**. A researcher is interested in the Twitter conversation around Australia’s federal election. The workflow for answering this problem would include preliminary feasibility check (to ensure there are sufficient data for analysis), collect, process, and understand the data, followed by analysis.
+
+To implement this workflow, we use the following tools:
+
+<ul>
+<li><a href="https://www.ado.eresearch.unimelb.edu.au/tutorials/" target="_blank">ADORed</a>: high-level social media analysis via interactive dashboard. It can be used to derive a list of tweet IDs relevant to the research question.</li>
```

Comment:
```suggestion
<li><a href="https://www.ado.eresearch.unimelb.edu.au/tutorials/" target="_blank">ADOReD</a>: high-level social media analysis via an interactive dashboard. It can be used to derive a list of tweet IDs relevant to the research question.</li>
```